Google-Summer-of-Code-2023.md

```diff
@@ -0,0 +1,272 @@
+## Table of Contents
+
+- [Getting started](#getting-started)
+- [FAQs](#faqs)
+- [Dates and Deadlines](#dates-and-deadlines)
+- [Types of work related to Oppia projects](#types-of-work-related-to-oppia-projects)
+- [GSoC proposal template](#gsoc-proposal-template)
+  - [Tips for writing a good project plan](#tips-for-writing-a-good-project-plan)
+  - [What should applicants expect from mentors in a proposal review?](#what-should-applicants-expect-from-mentors-in-a-proposal-review)
+  - [Sample proposals from past years](#sample-proposals-from-past-years)
+- [Selection Criteria](#selection-criteria)
+- [Communication](#communication)
+- [Oppia's Project Ideas List](#oppias-project-ideas-list)
+
+This year, Oppia will be participating in [Google Summer of Code 2023](https://summerofcode.withgoogle.com/)! GSoC is a global program which offers students and non-experienced contributors the opportunity to discover and work with open-source organizations. The contributions are supported by a stipend. Contributors work closely with one or more mentors from an open-source organization to implement either a project idea by the organization, or a proposal of their own. You might be interested in our GSoC info pages from previous years: [2022](https://github.com/oppia/oppia/wiki/Google-Summer-of-Code-2022), [2021](https://github.com/oppia/oppia/wiki/Google-Summer-of-Code-2021), [2020](https://github.com/oppia/oppia/wiki/Google-Summer-of-Code-2020), [2019](https://github.com/oppia/oppia/wiki/Google-Summer-of-Code-2019), [2018](https://github.com/oppia/oppia/wiki/Google-Summer-of-Code-2018), [2017](https://github.com/oppia/oppia/wiki/Google-Summer-of-Code-2017), [2016](https://github.com/oppia/oppia/wiki/Google-Summer-of-Code-2016).
+
+In order to receive updates about GSoC at Oppia, please subscribe to the [Oppia GSoC Announce](https://groups.google.com/g/oppia-gsoc-announce) mailing list.
+
+Oppia will be giving an automatic extension to all the projects i.e, all the projects will have 8 weeks (6 weeks for coding and 2 weeks for revision after evaluation) for both milestones. The coding period dates are strict and intended to balance offering flexibility to contributors with the need on the org admin side to have a streamlined process and a more time-bounded GSoC period.
```

Comment:
I think I would just say "...all the projects will have 8 weeks for each milestone, with an internal evaluation after 6 weeks" and drop the parenthetical. We can explain how it's divided later on but I wouldn't want to give the impression that "2 weeks for revision after evaluation" is standard -- most students ought to be done in 6 weeks and that should be the baseline expectation.

Additionally, maybe add "Contributors may opt for a shorter timeline if desired." at the end -- we don't want to cause a delay in stipends for contributors if they finish all their stuff well and on time.

Reply:
done. changed it